backend/profile/profile.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { allPublishedProfilesPayload } from './profile.service'
+
+export async function requestHandler(
```

Comment:
**nitpick:** This should not be a part of domain controller. Let's put it into `/backend/utils/http.utils.ts`